test/parallel/test-accessor-properties.js

```diff
@@ -33,27 +33,15 @@ const UDP = internalBinding('udp_wrap').UDP;
   }, TypeError);
 
   const StreamWrapProto = Object.getPrototypeOf(TTY.prototype);
+  const attributes = ['bytesRead', 'fd', '_externalStream'];
```

Comment:
Nit: maybe it's better to named as `properties` ?

Reply:
i’ll fix it

Reply:
resolved